hazelcast/src/main/java/com/hazelcast/client/impl/operations/GetConnectedClientsOperation.java

```diff
@@ -37,8 +38,9 @@ public GetConnectedClientsOperation() {
     @Override
     public void run() throws Exception {
         ClientEngineImpl service = getService();
-        this.clients = new HashMap<String, ClientType>();
-        for (Client clientEndpoint : service.getClients()) {
+        final Collection<Client> serviceClients = service.getClients();
+        this.clients = MapUtil.createHashMap(serviceClients.size());
```

Comment:
Maybe an idea to create a method 'createHashMapWithSize that accepts a collection as argument so you don't need to call 'size'? Anyhow; no need to change.